va/validation-authority_test.go

```diff
@@ -246,18 +244,19 @@ func TestSimpleHttpTLS(t *testing.T) {
 	port, err := getPort(hs)
 	test.AssertNotError(t, err, "failed to get test server port")
 	stats, _ := statsd.NewNoopClient()
-	va := NewValidationAuthorityImpl(&PortConfig{SimpleHTTPSPort: port}, stats, clock.Default())
+	va := NewValidationAuthorityImpl(&PortConfig{HTTPSPort: port}, stats, clock.Default())
```

Comment:
again, https://github.com/ietf-wg-acme/acme/pull/7 disallows TLS and this entire test function shall be removed